newsService/src/main/java/com/kovan/app/service/NewsService.java

```diff
@@ -51,23 +58,31 @@ public NewsService(RestTemplate restTemplate, ObjectMapper objectMapper,
         this.objectMapper = objectMapper;
         this.newsRepositoryService = newsRepositoryService;
     }
+
+    @Scheduled(cron = "0 0 */8 * * *")
```

Comment:
Add comments and try to explain the functionality

Reply:
Added comments.